doc/develop/west/zephyr-cmds.rst

```diff
@@ -174,3 +174,20 @@ As does deleting them::
 Additionally the tool allows you to specify the modules you want to list,
 fetch or clean blobs for by typing the module names as a command-line
 parameter.
+
+.. _west-twister:
+
+Twister wrapper: ``west twister``
+*****************************************
+This command is a wrapper for :ref:`twister <twister_script>`.  It leverages
+current :envvar:`ZEPHYR_BASE` environment variable and west manifest to launch
```

Comment:
done

Reply:
Unless there's a really compelling reason for making this separately configurable (even via ZEPHYR_BASE) that I'm missing, I'm reopening this comment to update the docs to just say that it uses the twister that's in the same repository as the twister command itself.

Reply:
I was giving capability to developer to be able to point to a different ZEPHYR_BASE, but I can see that also leading to problems.  I'll remove this capability.

Reply:
> I was giving capability to developer to be able to point to a different ZEPHYR_BASE, but I can see that also leading to problems.

The developer already has this capability though, is the thing. West respects ZEPHYR_BASE when looking for the workspace, so by the time the extension runs, the workspace has already been located in a way that respects this environment variable.  There doesn't seem to be a need for extra work here, because what can happen is that if you have `zephyr.base-prefer` set to `configfile` in your `.west/config` (see https://docs.zephyrproject.org/latest/develop/west/config.html#built-in-configuration-options), but ZEPHYR_BASE in the environment set to something else, you'll end up with a `west twister` implementation in workspace 1 that calls out to a twister implementation in workspace 2. It really doesn't make sense to me to do that because the twister APIs that the west extension is using are tightly coupled to what is in the current repository, and using another repository could break that. It's a footgun.

> I'll remove this capability.

Sounds good, thakns.